src/main/java/frc/robot/Config.java

```diff
@@ -95,8 +104,135 @@ else if (RobotBase.isSimulation()) {
 
    /** ADD CONSTANTS BELOW THIS LINE */
 
-  public class Swerve {
+  public static final Boolean swerveTuning = true;
+
+  public static final class Swerve {
+    public static final double stickDeadband = 0.1;
+ 
+    public static final int pigeonID = 30;
+    public static final boolean invertGyro = false; // Always ensure Gyro is CCW+ CW-
+
+    /* Drivetrain Constants Changed*/
+    public static final double trackWidth = Units.inchesToMeters(25.787);
+    public static final double wheelBase = Units.inchesToMeters(20.472);
+    public static final double wheelDiameter = Units.inchesToMeters(3.884);
+    public static final double wheelCircumference = wheelDiameter * Math.PI;
+
+    public static final double openLoopRamp = 0.25;
+    public static final double closedLoopRamp = 0.0;
+
+    public static final double driveGearRatio = (8.14 / 1.0); // 6.75:1
```

Comment:
Did we change the drive gear ratio?

Reply:
Oh that comment is leftover from the template code is my guess. We have L1 SDS mk4 modules which are 8.14 to 1 ratio.

![image](https://github.com/FRC2706/2024-2706-Robot-Code/assets/43829793/c5571ab6-4829-4e41-aa73-02d475bd7217)

Reply:
Remove the comment that says // 6.75:1